File: 724-find-pivot-index/find-pivot-index.java
class Solution {
    public int pivotIndex(int[] nums) {
        int sum = 0;
        int leftsum = 0;
        for( int i : nums){
            sum += i;
        }

        for(int i =0; i<nums.length; i++){
            if(leftsum == sum-nums[i]-leftsum){
                return i;
            }

            leftsum += nums[i];

        }

        return -1;
        
    }
}

File: 724-find-pivot-index/find-pivot-index.py
class Solution(object):
    def pivotIndex(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        sum = 0
        leftsum = 0
        for i in nums:
            sum += i

        for x in range(len(nums)):
            if(leftsum == sum - nums[x]-leftsum):
                return x
            leftsum += nums[x]
        
        return -1
        
        